frontend/src/tasks/LoadDailyCasesTask.js

```diff
@@ -4,7 +4,7 @@ class LoadDailyCasesTask {
   #getDailyCovidCases = async (datatype, iso_code) => {
     try {
       const res = await axios.get(
-        `http://localhost:8000/api/cases/${datatype}/daily/normalised/?iso_code=${iso_code}&end_date=2020-06-23`,
+        `http://localhost:8000/api/cases/${datatype}/daily/normalised/?iso_code=${iso_code}&end_date=2020-06-23&start_date=2020-02-20`,
```

Comment:
I have no problem with hardcoding start/end dates but I wonder whether there's a way we can do so in a central config file or similar. Not a fix needed for this PR but something to think about.

Reply:
Yeah, I agree. I'm working on it in PR #77, where the start and end dates are the ones corresponding to each country "first wave".